bin/ch-grow

```diff
@@ -505,12 +544,16 @@ def INFO(*args, **kwargs):
 def cmd(args, env=None):
    DEBUG("environment: %s" % env)
    DEBUG("executing: %s" % args)
-   INFO("\033[33m", end="")
+   color("33m", sys.stdout)
    cp = subprocess.run(args, env=env, stdin=subprocess.DEVNULL)
-   INFO("\033[0m", end="")
+   color("0m", sys.stdout)
    if (cp.returncode):
       FATAL("%s failed with return code %d" % (args[0], cp.returncode))
 
+def color(color, fp):
```

Comment:
Should the color stuff technically be in a different commit? I'm fine with keeping it here.

Reply:
Remember we squash-merge, so this will all become one commit.

You're right that it's a second concern and would be tidier in another PR. However, `ch-grow` output can't be tested without fixing #501, which is why it's here.

Reply:
Sounds good to me!